pom.xml

```diff
@@ -29,7 +29,7 @@
     <asm.version>6.2</asm.version>
     <jmh.version>1.20</jmh.version>
     <jmhjar.name>benchmarks</jmhjar.name>
-    <surefire.version>3.0.0-SNAPSHOT</surefire.version>
+    <surefire.version>2.22.0</surefire.version>
```

Comment:
oooh. 👍

Reply:
Does this work with jacoco?
Does this work with -T3 (currently disabled in this branch)?
Does this work with `@{argLine}` (yes, that's the correct syntax per the documentation and intellij support.)?

Reply:
jacoco: sure just have a look at the ci :-) https://jenkins.webtide.net/job/jetty.project/view/change-requests/job/PR-2436/
-T3 I can't see why it couldn't work. do you have any special log/error indicating not? (restored that and CI seems happy)
@{argLine} yup it works sorry I made test with a bad surefire snapshot version (restored and CI seems happy)